### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by MadhusudanTripathy**
This Is A New Feature

### CODE REVIEW

File: hello [hunk 1]
```diff
@@ -1 +1 @@
-version 3
\ No newline at end of file
+manually conflict resolved
```
Comment (mstoff77) [inline hello]: can be better

Comment (MadhusudanTripathy) [reply hello]: thanks now at the best

File: hello [hunk 2]
```diff
@@ -1 +1 @@
-version 3
```
Comment (mstoff77) [inline hello:1]: bug

File: hello [hunk 3]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+manually conflict resolved through pr comment
```
Comment (mstoff77) [inline hello:1]: bug

File: hello (hunk 2 above)
Comment (MadhusudanTripathy) [reply hello:1]: ok

File: hello (hunk 3 above)
Comment (MadhusudanTripathy) [reply hello:1]: ok

**Pull request merged**